### ISSUE TITLE
Separate application stack argument into multiple dynamic blocks

### ISSUE THREAD

**Issue by helenakallekleiv** [equinor/terraform-azurerm-function-app#55]
Terraform plan throws an `Conflicting configuration arguments` error when setting more than one application version type.

In this case, `use_dotnet_isolated_runtime` is false by default, which creates a conflict when setting e.g. `python_version`.

Error output:

```console
"site_config.0.application_stack.0.use_dotnet_isolated_runtime": conflicts with site_config.0.application_stack.0.python_version
```

**Comment by hknutsen** [comment]
We might be able to split the application stack object into many small simple type variables, for example:

```terraform
variable "application_stack_dotnet_version" {
  type = string
  default = null
}

variable "application_stack_use_dotnet_isolated_runtime" {
  type = string
  default = false
}

variable "application_stack_python_version" {
  type = string
  default = null
}
```

We'd have to implement custom validation rules to only allow configuration of a single `application_stack_*` variables at once.

Feel free to suggest better variable names.

**Comment by helenakallekleiv** [comment]
Fixed in #56
